db/migrate/20220411085523_remove_duration_and_review_method_from_commitments.rb

```diff
@@ -0,0 +1,7 @@
+class RemoveDurationAndReviewMethodFromCommitments < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :commitments, :duration, :string
+    remove_column :commitments, :review_method, :string
+    remove_column :commitments, :proposed_area_ha, :string
```

Comment:
If you're removing these columns, do we need the `ignore_column`s on the model?

Reply:
ignore_columns in the gem means if it sees a column in the csv it doesn't try to import it, so can either change the csv, or add this

Reply:
Ah I see!